common/djangoapps/third_party_auth/provider.py

```diff
@@ -39,7 +39,7 @@ def enabled(cls):
     @classmethod
     def accepting_logins(cls):
         """Returns list of providers that can be used to initiate logins currently"""
-        return [provider for provider in cls.enabled() if provider.accepts_logins]
+        return [provider for provider in cls.enabled() if provider.display_for_login]
```

Comment:
Makes sense. Previously, this method was essentially a 1:1 mapping with provider.accepts_logins, and was also a 1:1 mapping with displaying providers for login. It's no longer the first, so I think the best thing would probably be to rename it to better align with the second. Sound good?